CentralManagement/ChocoCCM Command Reference/Stop-CCMDeployment.md

```diff
@@ -0,0 +1,86 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Stop-CCMDeployment
+
+## SYNOPSIS
+Stops a running CCM Deployment
+

```

Comment:
I haven't looked at the implementation, but potentially problematic actions like this need a minimum of `ConfirmImpact="High"` in their cmdletbinding so folks are pretty much always prompted.